recipes-bsp/acpi-tables/acpi-tables.bb

```diff
@@ -18,7 +18,7 @@ SRC_URI = "\
 
 B = "${WORKDIR}/acpi-tables"
 
-inherit deploy
+inherit deploy allarch
```

Comment:
It is generally not recommended to use `allarch` when `RDEPENDS` is used ([link](https://www.yoctoproject.org/docs/2.6/ref-manual/ref-manual.html#ref-classes-allarch)), but the reasoning seems to be not applicable to our single-machine case.

What does this solve, anyway, I wonder?

Reply:
I doesn't solve anything. And I forgot about the RDEPENDS so that is wrong.